app.py

```diff
@@ -0,0 +1,46 @@
+from threads import *
+
+def cmd_help(args):
```

Comment:
For the help function, do you think it would be better to use the `argparse` module? I think it's a much easier way of adding help stuff for python command line. See `UIgnite/application.py` for a place I used it.